calendar_event_security_link_hr_holidays/models/hr_leave.py

```diff
@@ -0,0 +1,14 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models
+
+
+class HrLeave(models.Model):
+    _inherit = "hr.leave"
+
+    def action_refuse(self):
+        if self.env.user.has_group('hr.group_hr_user'):
```

Comment:
Reviewing this, I've noticed that my original requirements were wrong. Instead of enabling this for every HR Officer, it should be better using "All Approver" leaves group (`hr_holidays.group_hr_holidays_user`), as a more precise group for this, could you change it? Sorry!